bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/ComponentSensor.java

```diff
@@ -27,6 +30,7 @@
 @NonNullByDefault
 public class ComponentSensor extends AbstractComponent<ComponentSensor.ChannelConfiguration> {
     public static final String sensorChannelID = "sensor"; // Randomly chosen channel "ID"
+    private static final List<String> triggerIcons = Arrays.asList("mdi:toggle", "mdi:gesture");
```

Comment:
How about:
```
Pattern triggerIcons = Pattern.compile("^mdi:(toggle|gesture).*$");
```
and then
```
boolean trigger = triggerIcons.matcher(icon).matches();
```

Reply:
Thanks @jochen314 for your feedback! Changes done! :wink: